metricflow/sql/sql_bind_parameters.py

```diff
@@ -12,7 +13,7 @@ class SqlBindParameters(HashableBaseModel):
     These can be used as per https://docs.sqlalchemy.org/en/14/core/tutorial.html#using-textual-sql
     """
 
-    param_dict: Dict[str, str] = OrderedDict()
+    param_dict: Dict[str, SQLColumnType] = OrderedDict()
```

Comment:
Can we add test cases for non-string param types? Since we don't directly use the parameters in Metricflow itself there's no integration test coverage for this - we'll only know it breaks if an end user passes a parameterized query down.